packages/jest-cli/src/SearchSource.js

```diff
@@ -201,29 +201,38 @@ class SearchSource {
       ));
   }
 
+  static getTestSummary(patternInfo: PatternInfo) {
```

Comment:
It also contains the `getNoTestFound` message thing so I wanna colocate it because it operates on the same data. I agree though that we should probably put these things somewhere else eventually.